### DISCUSSION TITLE
Clarification Needed on Equivalent Variant Pairs

### DISCUSSION THREAD

**Discussion by ghost** [ORTools4MTEX/ORTools#75] [Q&A] [answered]
Hi everyone,

Could someone please explain what "equivalent variant pairs" means? for example why we consider (V1-V2/V3-V4/V5-V6) as equivalent variant pairs?
Specifically, I would like to understand the following:
1. Do equivalent variant pairs have the same misorientation?
2. Are they part of the same Bain/Packet group?
3. Do they have the same rotation matrix? or habit plane?

Thank you!

**Comment by frankNiessen** [comment]
Hi, 

in our example 10, case 3 concerns equivalent variant pairs. Here we have attached a reference that very well explains why these pairs are equivalent: 

```
%% OPTION 3: Equivalent variant id child grain pair analysis
screenPrint('Step','Equivalent (or other) variant id child grain pair analysis');
% Calculate equivalent variant Ids of martensitic variants (block
% boundaries) in steel microstructures as per the analysis in the
% following reference:
%
% S. Morito, A.H. Pham, T. Hayashi, T. Ohba, Block boundary analyses to
% identify martensite and bainite, Mater. Today Proc., Volume 2,
% Supplement 3, 2015, Pages S913-S916.
% (https://doi.org/10.1016/j.matpr.2015.07.430)
%
```

**Reply by ghost** [reply]
Thank you for your help. 
In the paper, it states that "The block boundaries are classified into four types." When plotting the distribution of variant pairs, pairs such as V1-V9 and V1-V19 are considered equivalent which both have same misorientation (50.51 degrees). In addition, V1-V10 and V1-V14 are considered equivalent too which have same misorientation with V1-V9/V1-V19 but they are in different group. Could you please explain why these pairs are not treated as equivalent? and Are they classified as block boundaries? 
![Variant Pairs with V1](https://github.com/user-attachments/assets/9b439ddc-6016-4849-a487-592bf90e9dce)

**Comment by frankNiessen** [comment, marked as answer]
Hi again, 

if you have a look at Table 2 in reference 

```
Morito, S., Tanaka, H., Konishi, R., Furuhara, T., & Maki, T. (2003). The morphology and crystallography of lath martensite in Fe-C alloys. Acta Materialia, 51(6), 1789–1799. https://doi.org/10.1016/S1359-6454(02)00577-3
```

you will be able to see that V1-V10 / V1-V14 groups may have the same misorientation angle as the V1-V9/V1-V19 group, but a different misorientation axis. For misorientations between two crystal orientations to be equivalent, both the misorientation angle **and** axis must be equivalent.

Best wishes
Frank
